src/ui/stats-container.ts

```diff
@@ -1,7 +1,8 @@
 import BBCodeText from "phaser3-rex-plugins/plugins/gameobjects/tagtext/bbcodetext/BBCodeText";
 import BattleScene from "../battle-scene";
-import { Stat, getStatName } from "../data/pokemon-stat";
 import { TextStyle, addBBCodeTextObject, addTextObject, getTextColor } from "./text";
+import { getStatKey, PERMANENT_STATS } from "#app/enums/stat.js";
```

Comment:
```suggestion
import { PERMANENT_STATS, getStatKey } from "#app/enums/stat.js";
```
NIT: Label types and standardize order